docs/spec/channel.md

```diff
@@ -187,13 +187,19 @@ exclusively communicate using CloudEvents.
 #### Input
 
 Every Channel MUST expose either an HTTP or HTTPS endpoint. It MAY expose both.
-The endpoint(s) MUST conform to CloudEvents
-[Version 1.0](https://github.com/cloudevents/spec/blob/v1.0/http-protocol-binding.md).
+The endpoint(s) MUST conform to one of the following versions of the specification:
+
+* [CloudEvents 0.2 specification](https://github.com/cloudevents/spec/blob/v0.2/http-transport-binding.md)
+* [CloudEvents 0.3 specification](https://github.com/cloudevents/spec/blob/v0.3/http-transport-binding.md)
+* [CloudEvents 1.0 specification](https://github.com/cloudevents/spec/blob/v1.0/http-protocol-binding.md)
+
 The Channel MUST NOT perform an upgrade of the passed in version. It MUST emit
-the event with the same version. It MUST support both Binary Content mode and
-Structured Content mode. The HTTP(S) endpoint MAY be on any port, not just the
-standard 80 and 443. Channels MAY expose other, non-HTTP endpoints in addition
-to HTTP at their discretion (e.g. expose a gRPC endpoint to accept events).
+the event with the same version. It MUST support both _Binary Content Mode_ or
```

Comment:
this seems wrong? It used to say must support BOTH and, but not it says BOTH or. Does it need to support both (like it said before)?

Reply:
ah ... _and_ is the trick :-)